main/com/collections/Methods.java

```diff
@@ -62,34 +62,75 @@ public boolean checkIfAnagram(String first, String second) {
 
     public boolean canBePalindrome(String s) {
         List<Character> word = new ArrayList<>();
-        List<Character> wordCopy= new ArrayList<>();
+        List<Character> wordCopy = new ArrayList<>();
 
         //adding single characters as an elements of Lists
         for (char c : s.toCharArray()) {
             word.add(c);
             wordCopy.add(c);
         }
 
-        int nonPairsCharacters=0;
+        int nonPairsCharacters = 0;
         int counter = 0;
 
-        while(!(word.isEmpty())){
+        while (!(word.isEmpty())) {
 
             //counting of quantity every type of character in the word
-            while (word.contains(wordCopy.get(0))){
-               counter++;
+            while (word.contains(wordCopy.get(0))) {
+                counter++;
                 word.remove(wordCopy.get(0));
             }
             //counting of non pairs characters
-            if(counter%2==1){
+            if (counter % 2 == 1) {
                 nonPairsCharacters++;
             }
             //update wordCopy list
             wordCopy = List.copyOf(word);
 
             //set up counter to 0 before next loop
-            counter=0;
+            counter = 0;
         }
         return nonPairsCharacters == 1 || nonPairsCharacters == 0;
     }
+
+
+    public int[] sumTwoIntArrays(int[] array1, int[] array2) {
+
+        //Convert array to string builder
+        StringBuilder stringBuilder1 = new StringBuilder();
+        StringBuilder stringBuilder2 = new StringBuilder();
+        for (int item : array1) {
```

Comment:
I think you can either:
- assume that all digits in array are less than 10 and remove below 'if',
- don't assume this but throw exception and terminate program if digit is greater than 9